### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by Denis2626**

### CODE REVIEW

**Review by wizeguyy — COMMENTED** [review]
@jdowning100 will have to review this. This touches too much that I am not familiar enough with. I can at least say:

Please squash your commits into atomic changes with meaningful commit messages. No merge commits, please.

**Review by wizeguyy — DISMISSED** [review]
Please do not include merge commits.
Please make your commits atomic.

**Comment by jdowning100** [comment]
Hey @DenisIvanov26 @wizeguyy I noticed he squashed in my previously merged commit https://github.com/PelagusWallet/pelagus-extension/commit/ee8c11c0ed61ed9ba2ae4995b9e6ce2457f8bdba
Is this a problem?

File: ui/pages/Send.tsx [hunk 1]
```diff
@@ -260,10 +260,10 @@ export default function Send(): ReactElement {
...
               }
             />
-            {assetType === "token" && !hasError && (
-              <div className="value">
-                ${assetAmount?.localizedMainCurrencyAmount ?? "-"}
-              </div>
+            {assetType === "token" && !hasError && ( null
+              // <div className="value">
+              //   ${assetAmount?.localizedMainCurrencyAmount ?? "-"}
+              // </div>
```
Comment (jdowning100) [inline ui/pages/Send.tsx:266]: I would prefer you remove the comments tbh

File: ui/pages/SingleAsset.tsx [hunk 2]
```diff
@@ -177,9 +177,9 @@ export default function SingleAsset(): ReactElement {
               )}
             </div>
             <div className="balance">{localizedDecimalAmount}</div>
-            {localizedMainCurrencyAmount && (
+            {/* {localizedMainCurrencyAmount && (
```
Comment (jdowning100) [inline ui/pages/SingleAsset.tsx:180]: Again just delete unused code, don't comment it out

**Review by jdowning100 — APPROVED** [review]
The code looks good. However I would like Clark, Mason and Raleigh to have a look at the wallet before we merge. I was under the impression that Raleigh wanted a black-on-white design instead of the current grey(ish)-on-blue.
Also, in the Add Custom Asset page the contract address label in the text box is greyed out.

File: ui/pages/Send.tsx (hunk 1 above)
Comment (Denis2626) [reply ui/pages/Send.tsx:266]: I think it is fine leaving them commented out as it will be easier to add the feature back once mainnet comes around

**Review by robschleusner — APPROVED** [review]

**Pull request merged**